_data/jobs.yml

```
@@ -1,6 +1,9 @@
 - expires: 2021-09-30
-  location: Rand Corporation, Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington,
-    D.C.
+  location: Rand Corporation: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington, DC
```

I think it might be getting thrown off by the multiple :'s here
```suggestion
  location: "Rand Corporation: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington, DC"
```